packages/karma-runner/src/karma-plugins/test-hooks-middleware.ts

```diff
@@ -88,6 +88,7 @@ export class TestHooksMiddleware {
       case 'jasmine':
         this.currentTestHooks = `
       window.${SHOULD_REPORT_COVERAGE_FLAG} = true;
+      window.__stryker__ = window.__stryker__ || {};
```

Comment:
```suggestion
      window.${NAMESPACE} = window.${NAMESPACE} || {};
```

Reply:
Also, the same should be done for mocha below 👇.

Ideally, we should declare this part above and reuse it for both `mocha` and `jasmine`.

Reply:
changed `__stryker__` to `${NAMESPACE }`, moved the line , together with 
`window.${SHOULD_REPORT_COVERAGE_FLAG} = true;` before the `switch` so that the change should work for mocha, too